script/index.ts

```diff
@@ -63,6 +63,13 @@ function buildOptionsForLanguage(language: Language, beautifiers: String[]) {
       type: "string"
     }
   }
+  languageOptions["beautify_on_save"] = {
```

Comment:
Thinking about this more and more and I feel like these kind of decisions should be User and/or Editor specific. Storing in a file means this is a project level concern. While selected beautifiers and options make sense to project level, enforcing WHEN I beautify such as `beautify_on_save` feels like the User should be personally setting this in their editor. For instance, I do NOT use beautify on save. I run it manually when I want it. However, looking at the stats, most people prefer the beautify on save option to be enabled.

Thoughts? I do not want to remove this from PR, just discuss it :).

Reply:
It is for the editor, it's the script that generates the options.json file for Atom Beautify's settings.

Reply:
Nevermind. I thought this was for extracting from the configuration file.

Although I do want to have a discussion in a separate thread about the above and what options should be supported in the configuration file.